src/contracts/interfaces/IBLSRegistryCoordinatorWithIndices.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "./IRegistryCoordinator.sol";
+import "./IStakeRegistry.sol";
+import "./IBLSPubkeyRegistry.sol";
+import "./IIndexRegistry.sol";
+
+/**
+ * @title Minimal interface for the `IBLSStakeRegistryCoordinator` contract.
+ * @author Layr Labs, Inc.
+ */
+interface IBLSRegistryCoordinatorWithIndices is IRegistryCoordinator {
+    // STRUCTS
+
+    struct QuorumBitmapUpdate {
+        uint32 updateBlockNumber;
+        uint32 nextUpdateBlockNumber;
+        uint192 quorumBitmap;
+    }
+
+    struct OperatorSetParam {
+        uint32 maxOperatorCount;
+        uint8 kickPercentageOfOperatorStake;
+        uint8 kickPercentageOfAverageStake;
+        uint8 kickPercentageOfTotalStake;
```

Comment:
seems a little coarse-grained to use percentages here, might prefer BIPs at least (and it should still fit in a single slot)

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/d6e5a76d940683844f264a1748cfa584fb0e60d4